third_party/move/move-compiler/tests/move_check/deprecated/public_script.exp

```diff
@@ -0,0 +1,12 @@
+warning[W00001]: DEPRECATED. will be removed
+  ┌─ tests/move_check/deprecated/public_script.move:2:5
+  │
+2 │     public(script) fun main() {
+  │     ^^^^^^^^^^^^^^ 'public(script)' is deprecated in favor of the 'entry' modifier. Replace with 'public entry'
+
+error[E03001]: address with no value
+  ┌─ tests/move_check/deprecated/public_script.move:3:30
+  │
+3 │         let _addr:address = @Test;
+  │                              ^^^^ address 'Test' is not assigned a value. Try assigning it a value when calling the compiler
```

Comment:
Any idea how this test is different than third_party/move/move-compiler/tests/move_check/parser/function_visibility_script.move ?  Why does this one trigger the assertion?

Reply:
This assertion will cause the compiler to panic when the target move code contains public (script) functions and there are other errors at or above NonblockingError level. This example injects an error.

Reply:
Ideally, you might add a comment to indicate this:
```
// Previously, deprecation plus an error led to a compiler assert.  Make sure that doesn't come back.
```